src/index.html

```diff
@@ -11,12 +11,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
```

Comment:
Missing BEM class:

Add a class for this img based on your choice from the previous comment:
(considering img as "logo-image")
 1. **logo** is an _Element_ of the _Block_ **header**:
 ```
<a
  class="header__logo-image"
  href="index.html"
>
```
 2. **logo** is a _Block_, but also child of the **header** _Block_:
  ```
<a
  class="logo__logo-image"
  href="index.html"
>
```